uk.co.compucorp.civicrm.hrleaveandabsences/js/angular/test/shared/models/entitlement-model_test.js

```diff
@@ -0,0 +1,86 @@
+define([
+  'leave-absences/shared/apis/entitlement-api',
```

Comment:
if i remove them i get this error 

`
Error: [$injector:unpr] http://errors.angularjs.org/1.5.9/$injector/unpr?p0=EntitlementInstanceProvider%20%3C-%20EntitlementInstance%20%3C-%20Entitlement
`